infra-as-code/bicep/modules/vwanConnectivity/parameters/mc-vwanConnectivity.parameters.min.json

```diff
@@ -2,41 +2,44 @@
   "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
+    "parLocation": {
+      "value": "chinaeast2"
+    },
+    "parCompanyPrefix": {
+      "value": "alz"
+    },
+    "parVirtualHubAddressPrefix": {
+      "value": "10.100.0.0/23"
+    },
+    "parAzureFirewallTier": {
+      "value": "Standard"
+    },
     "parVirtualHubEnabled": {
       "value": true
     },
-    "parVPNGatewayEnabled": {
+    "parVpnGatewayEnabled": {
       "value": true
     },
-    "parERGatewayEnabled": {
+    "parExpressRouteGatewayEnabled": {
       "value": true
     },
     "parAzureFirewallEnabled": {
       "value": true
     },
-    "parNetworkDNSEnableProxy": {
+    "parNetworkDnsEnableProxy": {
       "value": true
     },
-    "parDdosEnabled": {
-      "value": false
-    },
-    "parDdosPlanName": {
-      "value": "alz-ddos-plan"
-    },
-    "parCompanyPrefix": {
-      "value": "alz"
+    "parAzureFirewallAvailabilityZones": {
+      "value": []
     },
-    "parPublicIPSku": {
-      "value": "Standard"
+    "parVpnGatewayScaleUnit": {
+      "value": 1
     },
-    "parVhubAddressPrefix": {
-      "value": "10.100.0.0/23"
+    "parExpressRouteGatewayScaleUnit": {
+      "value": 1
     },
-    "parAzureFirewallTier": {
-      "value": "Standard"
-    },
-    "parAzureFirewallAvailabilityZones": {
-      "value": []
+    "parDdosEnabled": {
+      "value": true
```

Comment:
Mooncake does not support DDoS

Reply:
Just pushed a change for all instances of `parDdosEnabled` in mc- parameters files.